_workinggroups/softwaredevelopertools.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Software/Developer Tools 
+layout: plain
+redirect_from:
+  - /activities/softwaredevelopertools.html
+---
+
+The Software/Developer Tools Working Group aims to help HEP community in developing the latest and modern feature-rich projects
```

Comment:
The Software/Developer Tools Working Group aims to help **the** HEP community in developing the latest **,** ~~and~~ modern feature-rich projects

Reply:
Done.